cmd/jsonnetfmt/cmd.go

```diff
@@ -38,28 +38,34 @@ func usage(o io.Writer) {
 	fmt.Fprintln(o, "jsonnetfmt {<option>} { <filename> }")
 	fmt.Fprintln(o)
 	fmt.Fprintln(o, "Available options:")
-	fmt.Fprintln(o, "  -h / --help             This message")
-	fmt.Fprintln(o, "  -e / --exec             Treat filename as code")
-	fmt.Fprintln(o, "  -o / --output-file <file> Write to the output file rather than stdout")
-	fmt.Fprintln(o, "  -c / --create-output-dirs  Automatically creates all parent directories for files")
-	fmt.Fprintln(o, "  -i / --in-place         Update the Jsonnet file(s) in place.")
-	fmt.Fprintln(o, "  --test                  Exit with failure if reformatting changed the file(s).")
-	fmt.Fprintln(o, "  -n / --indent <n>       Number of spaces to indent by (default 2, 0 means no change)")
-	fmt.Fprintln(o, "  --max-blank-lines <n>   Max vertical spacing, 0 means no change (default 2)")
-	fmt.Fprintln(o, "  --string-style <d|s|l>  Enforce double, single (default) quotes or 'leave'")
-	fmt.Fprintln(o, "  --comment-style <h|s|l> # (h), // (s)(default), or 'leave'; never changes she-bang")
-	fmt.Fprintln(o, "  --[no-]pretty-field-names Use syntax sugar for fields and indexing (on by default)")
-	fmt.Fprintln(o, "  --[no-]pad-arrays       [ 1, 2, 3 ] instead of [1, 2, 3]")
-	fmt.Fprintln(o, "  --[no-]pad-objects      { x: 1, y: 2 } instead of {x: 1, y: 2} (on by default)")
-	fmt.Fprintln(o, "  --[no-]sort-imports     Sorting of imports (on by default)")
-	fmt.Fprintln(o, "  --version               Print version")
+	fmt.Fprintln(o, "  -h / --help                This message")
+	fmt.Fprintln(o, "  -e / --exec                Treat filename as code")
+	fmt.Fprintln(o, "  -o / --output-file <file>  Write to the output file rather than stdout")
+	fmt.Fprintln(o, "  -c / --create-output-dirs  Automatically creates all parent directories for")
+	fmt.Fprintln(o, "                             files")
+	fmt.Fprintln(o, "  -i / --in-place            Update the Jsonnet file(s) in place")
+	fmt.Fprintln(o, "  --test                     Exit with failure if reformatting changed the")
+	fmt.Fprintln(o, "                             file(s)")
+	fmt.Fprintln(o, "  -n / --indent <n>          Number of spaces to indent by")
+	fmt.Fprintln(o, "                             (default 2, 0 means no change)")
```

Comment:
not really sure whether this is better than breaking in the middle of the "default" thingy

Reply:
It's ok.